### PR TITLE
Path config

### PR DESCRIPTION
**Pull request by vuillaut**

### CODE REVIEW

**Comment by vuillaut** [comment]
I believe this is ready to be merged in the refactor branch so you can use it @garciagenrique

File: lstmcpipe/config/tests/test_config.py [hunk 1]
```diff
@@ -1,119 +1,6 @@
...
-    assert parsed_config["stages_to_run"] == ["r0_to_dl1"]
-    assert parsed_config["workflow_kind"] == "lstchain"
-
-    new_dummy = config.copy()
-    new_dummy["slurm_config"] = {"user_account": "aswg"}
-    new_parsed_config = parse_config_and_handle_global_vars(new_dummy)
-    assert new_parsed_config["batch_config"]["slurm_account"] == "aswg"
+def test_base_config():
+    cfg = config.base_config()
+    assert "source_environment" in cfg
```
Comment (garciagenrique) [inline lstmcpipe/config/tests/test_config.py:6]: Could you add all `workflow_kind` and `prod_id` (and I would say that `slurm config` too

**Review by garciagenrique — COMMENTED** [review]
Nothing against of changing the yaml lib, but which are the exact needs of this change ?

**Comment by garciagenrique** [comment]
Looks nice otherwise !  Merging to branch in the moment we resolve the `test_config` conversation

File: lstmcpipe/config/tests/test_config.py (hunk 1 above)
Comment (vuillaut) [reply lstmcpipe/config/tests/test_config.py:6]: in the test?
sure

**Comment by vuillaut** [comment]
> Nothing against of changing the yaml lib, but which are the exact needs of this change ?

Ha yes good question.
`ruamel.yaml` is much more powerful and is able to handle the order of the loaded config as well as keeping comments, both things that are needed imo.

File: lstmcpipe/config/tests/test_config.py (hunk 1 above)
Comment (vuillaut) [reply lstmcpipe/config/tests/test_config.py]: Done

**Review by garciagenrique — APPROVED** [review]

**Pull request merged**